dumBash.sh

```diff
@@ -91,18 +91,40 @@ function command_not_found_handler () {
 	    case "${ASK}" in
 		    [Yy][Ee][Ss]|Y|y|[غِ][ثٍ][سئ]|[غِ][سئ]|ص|ب|د|صحیح|بله|درست|یس|غ|ِ)
 			    logger "GREEN" "Running: " "nobreak"
-			    logger "" "${NEW}..."
+			    logger "" "${NEW}"
+			    if [ "${NEW}" == "exit" ]; then
+				    TEMP_VAR="$(ps -p $(ps -p $$ -o ppid=))"
+				    TEMP_VAR="$(printf "%s" "${TEMP_VAR}" | awk 'NR==2 {print $1}')"
+				    kill -s TERM "${TEMP_VAR}"
+			    fi
 			    eval $NEW
-			    if [ "$?" != 0 ];then
+			    if [ "$?" -ne 0 ];then
 				    return 1
 			    fi
 			    ;;
+
+		    '')
+        		    logger "GREEN" "Running: " "nobreak"
+        		    logger "" "${NEW}"
+        		    if [ "${NEW}" == "exit" ]; then
+                		    TEMP_VAR="$(ps -p $(ps -p $$ -o ppid=))"
+                		    TEMP_VAR="$(printf "%s" "${TEMP_VAR}" | awk 'NR==2 {print $1}')"
+                		    kill -s TERM "${TEMP_VAR}"
+        		    fi
+        		    eval $NEW
+        		    if [ "$?" -ne 0 ]; then
+                		    return 1
+        		    fi
+        		    ;;
+
```

Comment:
This has the same functionality that last statement had, in 
```
		    [Yy][Ee][Ss]|Y|y|[غِ][ثٍ][سئ]|[غِ][سئ]|ص|ب|د|صحیح|بله|درست|یس|غ|ِ)
```
That `|)` is meant to be `|'')` which does the same functionality as what you added here.
Any reasons you did it in another case statement rather than putting it in the last case?

Reply:
Nope.
You can test your own.
(`|)`) i mean.